docs/formats.md

```diff
@@ -103,12 +103,12 @@ Since static deltas may not exist, the client first needs to attempt
 to locate one.  Suppose a client wants to retrieve commit `${new}`
 while currently running `${current}`.
 
-The first thing to understand is that in order to save space, these
+The first thing to understand is that, in order to save space, these
```

Comment:
I don't think this comma is correct.  Maybe simpler to just eliminate the whole clause since it's "filler".  IOW just say
"In order to save space, ...".

Reply:
To build on https://owl.purdue.edu/owl/general_writing/punctuation/commas/extended_rules_for_commas.html - the rationale for base64 is almost entirely space saving, so having it be "comma quoted" like this doesn't seem right.

Reply:
I followed Rule 3 of https://owl.purdue.edu/owl/general_writing/punctuation/commas/extended_rules_for_commas.html.
But I agree that we could just remove the filler.